resources/views/home.blade.php

```diff
@@ -2,19 +2,24 @@
     @guest
         @include('content.welcome')
     @endguest
-    <x-news.carousel/>
-    <?php
-    renderFinishedClaimsComponent(5);
-    RenderActivePlayersComponent();
-    ?>
-    <x-user.online-count-chart/>
-    <?php
-    renderNewClaimsComponent(5);
-    RenderRecentForumPostsComponent();
-    ?>
+
+    <x-news.carousel />
+    <x-claims.finished-claims count="6" />
```

Comment:
I'm still only seeing 5 rows.

Reply:
I am unable to reproduce this and continue to see 6 on my local.

![Screenshot 2023-03-20 at 5 47 35 PM](https://user-images.githubusercontent.com/3984985/226473634-f9d20bf8-15f0-4f72-9c2a-dfb3affb67eb.png)

Is it possible something is awry with your local environment / build?

Reply:
I've got one item being hidden by #1409. Your implementation doesn't fetch another to fill the void.

Reply:
Ah! I understand now.

In https://github.com/RetroAchievements/RAWeb/pull/1425/commits/14f4b5d1023eb834da4600e580cd8031b498a3b9, I've reimplemented the original do/while logic from _set-claim.php_ and have done a small refactor to get it down to a simple `while()` loop. This should resolve the issue.